db/schema.rb

```diff
@@ -82,8 +82,10 @@
     t.boolean "forbid_newcomers", default: false, null: false
     t.string "forbid_newcomers_reason"
     t.integer "registration_version", default: 0, null: false
-    t.integer "competitor_can_cancel", default: 0, null: false
     t.integer "newcomer_month_reserved_spots"
+    t.boolean "auto_accept_registrations", default: false, null: false
+    t.integer "auto_accept_disable_threshold", default: 0, null: false
+    t.integer "competitor_can_cancel", default: 0, null: false
```

Comment:
Why is this column jumping to a different position? Feels like merge-whoopsie

Reply:
Does the order here matter? I just removed the merge markers and committed the resulting order